### DISCUSSION TITLE
Video Documentation

### DISCUSSION THREAD

**Discussion by Birkis** [frzi/swiftui-router#18] [Q&A]
A great idea that solves one of the biggest issues with SwiftUI! However, I found the documentation a little difficult to follow. Would it be possible to publish a sample project using a basic implementation of the router? And/or a video introduction? 😬

**Comment by frzi** [comment]
At one point I was working on an example project showing the very basics of SwiftUI Router, but eventually started using the package for a production project (which lead to making the changes for version 1.0.0). A short tutorial/introduction on Medium was also planned.

Unfortunately, time and priorities (and other SwiftUI projects!) have put me a bit off course. 😬  When given the chance I will gladly work on an example project and tutorial again. In the meantime, if someone else has any resources for something similar, I'll gladly link them in the readme. 

Terribly sorry, as I completely understand SwiftUI Router can be a bit of maze when using it for the first time. If there are any questions regarding SwiftUI Router I will do my absolute best to answer them.
